Exporter/src/main/java/uk/ac/ebi/biostd/exporter/configuration/GeneralConfiguration.java

```diff
@@ -43,4 +46,9 @@ public RemoteService remoteService(@Value("${jobs.backend-url}") String backendU
     public MagicFolderUtil magicFolderUtil(UsersFoldersProperties properties) {
         return new MagicFolderUtil(properties.getBaseDropboxPath(), properties.getSymLinksPath());
     }
+
+    @Bean
+    public List<String> libFileProjects() {
```

Comment:
this type of bean really dangeorus, another part of the code can easly try to inject a bean of type `List<String>` we need to inject values directly in `SubmissionService`